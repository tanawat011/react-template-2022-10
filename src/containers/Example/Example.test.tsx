import { render, screen } from '@testing-library/react'

import { Example } from './Example'

describe('<Example />', () => {
  test('should render', () => {
    render(<Example />)
    const linkElement = screen.getByText(/learn react/i)
    expect(linkElement).toBeInTheDocument()
  })

  test('match snapshot', () => {
    const { asFragment } = render(<Example />)
    expect(asFragment()).toMatchSnapshot()
  })
})
