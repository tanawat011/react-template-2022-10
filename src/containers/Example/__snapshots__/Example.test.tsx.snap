// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Example /> match snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <header
      class="App-header"
    >
      <img
        alt="logo"
        class="App-logo"
        src="logo.svg"
      />
      <p
        class="bg-red-700"
      >
        Edit 
        <code>
          src/container/Example.tsx
        </code>
         and save to reload.
      </p>
      <a
        class="App-link"
        href="https://reactjs.org"
        rel="noopener noreferrer"
        target="_blank"
      >
        learn react
      </a>
      <a
        href="/about"
      >
        About
      </a>
      <code
        style="background: green;"
      >
        {"name":"","age":0}
      </code>
      <code
        style="background: blue;"
      >
        {"id":2,"name":"doe"}
      </code>
    </header>
  </div>
</DocumentFragment>
`;
